### PR TITLE
Project page update

### PR DESCRIPTION
**Pull request by zsoltk**

### CODE REVIEW

File: documentation/how-to-use-appyx/quick-start.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Quick start guide
 
+!!! info
```
Comment (LachlanMcKee) [inline documentation/how-to-use-appyx/quick-start.md:3]: Do these render special characters when using the doc tool?

File: documentation/news.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# News & updates
```
Comment (LachlanMcKee) [inline documentation/news.md:1]: When do we need to update this file?

File: documentation/how-to-use-appyx/quick-start.md (hunk 1 above)
Comment (zsoltk) [reply documentation/how-to-use-appyx/quick-start.md:3]: Yep, https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types

File: documentation/news.md (hunk 2 above)
Comment (zsoltk) [reply documentation/news.md:1]: New codelabs, new pages to documentation, announcements, or bigger updates to the project that would either not be covered by changelog (e.g. codelabs is external), or noteworthy enough to not be lost among a 100 other changes.

File: documentation/how-to-use-appyx/quick-start.md (hunk 1 above)
Comment (LachlanMcKee) [reply documentation/how-to-use-appyx/quick-start.md:3]: sounds good!

File: documentation/news.md (hunk 2 above)
Comment (LachlanMcKee) [reply documentation/news.md:1]: so for every PR we need to potentially keep this in mind?

**Review by LachlanMcKee — APPROVED** [review]

**Pull request merged**